docs/pywbemclicommands.rst

```diff
@@ -1009,7 +1008,7 @@ The commands in this group are:
 Connection add command
 ^^^^^^^^^^^^^^^^^^^^^^
 
-The ``connection add`` command creates a new connection using the command arguments and sets the new
+The ``connection add`` command creates a new connection using the command arguments and options and sets the new
 connection as the current connection. This command saves the
```

Comment:
The text "and sets the new connection as the current connection." is likely incorrect, given the description of "connection add" that claims the opposite.

Reply:
You are right. DONE